sdk/storage/Azure.Storage.Files.DataLake/src/DataLakeFileClient.cs

```diff
@@ -2799,7 +2799,9 @@ public new DataLakeFileClient WithCustomerProvidedKey(Models.DataLakeCustomerPro
         /// A <see cref="RequestFailedException"/> will be thrown if
         /// a failure occurs.
         /// </remarks>
+#pragma warning disable AZC0002 // Client method should have an optional CancellationToken (both name and it being optional matters) or a RequestContext as the last parameter.
```

Comment:
These look like valid exemptions - do we know why the analyzer didn't catch this one before?